app/src/util/tools/other.ts

```diff
@@ -0,0 +1,223 @@
+import axios from 'axios'
+import Big from 'big.js'
+
+import {
+  CONFIRMATION_COUNT,
+  MAIN_NETWORKS,
+  RINKEBY_NETWORKS,
+  SOKOL_NETWORKS,
+  XDAI_NETWORKS,
+} from '../../common/constants'
+import { MarketTokenPair } from '../../hooks/useGraphMarketsFromQuestion'
+import { CompoundService } from '../../services'
+import { getNativeAsset, getWrapToken, networkIds } from '../networks'
+import { BalanceItem, Token, TransactionStep } from '../types'
+
+export const checkRpcStatus = async (customUrl: string, setStatus: any, network: any) => {
```

Comment:
Just an idea that we don't have to include in this PR - The `utils/networks.ts` file was originally setup to contain functions for accessing information about what network the user is on, over time I added utility functions like `getGraphMeta`, `getBySafeTx`,  and `waitForBlockToSync` that don't really fit in there.

We could potentially move those functions into a new `utils/tools/network` or `utils/tools/http` file along with `checkRpcStatus` and `isValidHttpUrl`. Thoughts?

Reply:
I like the idea! I also think the base util folder could be sorted more! Will implement it!